docs/content/tutorial/index.ngdoc

```diff
@@ -62,24 +65,26 @@ a few git commands.
 ### Install Git
 
 You can download and install Git from http://git-scm.com/download. Once installed, you should have
-access to the `git` command line tool.  The main commands that you will need to use are:
+access to the `git` command line tool. The main commands that you will need to use are:
+
+* `git clone ...`: Clone a remote repository onto your local machine.
+* `git checkout ...`: Check out a particular branch or a tagged version of the code to hack on.
```

Comment:
There is an annoying issue about corporate firewalls blocking git

https://github.com/angular/angular-phonecat/issues/141
and probably
https://github.com/angular/angular-phonecat/issues/235

Can we add this information into the guide?